src/uk/gov/hmcts/contino/azure/Acr.groovy

```diff
@@ -99,6 +102,34 @@ class Acr extends Az {
     this.run()
   }
 
+  def reconcile(DockerImage dockerImage) {
+    String repository = dockerImage.getRepositoryName().replace("/", "-")
+    def kubectl = new Kubectl(this, subscription, namespace, params.aksSubscription.name)
+    kubectl.loginJenkins()
+    steps.sh (
+      script: "kubectl config get-contexts",
```

Comment:
I think this would be simpler if you just wrote a single shell script that you can test locally and then load it with:
https://github.com/hmcts/cnp-jenkins-library/blob/5d1835af2554f61489c9a3e9acf3018e94a5f287/vars/warnAboutDeprecatedChartConfig.groovy#L10

and store it in the corresponding place under https://github.com/hmcts/cnp-jenkins-library/tree/master/resources/uk/gov/hmcts